src/bacnet/basic/service/h_apdu.c

```diff
@@ -53,6 +53,7 @@
 static uint16_t Timeout_Milliseconds = 3000;
 /* Number of APDU Retries */
 static uint8_t Number_Of_Retries = 3;
+static int Local_Network_Priority; /* Fixing test 10.1.2 Network priority */
```

Comment:
We should store the smaller uint8_t datatype that we use in the get/set.

Reply:
Fixed